db/schema.sql

```diff
@@ -28,7 +25,7 @@ CREATE TABLE task_aggregator_auth_keys(
     ord BIGINT NOT NULL,      -- a value used to specify the ordering of the authentication keys
     key BYTEA NOT NULL,       -- HMAC key used to authenticate messages to/from the other aggregator (encrypted)
 
-    CONSTRAINT unique_task_id_and_ord UNIQUE(task_id, ord),
+    CONSTRAINT auth_key_unique_task_id_and_ord UNIQUE(task_id, ord),
```

Comment:
Q: do these table-scoped constraints need to be uniquely named across the entire DB?

Reply:
Yes, I got an error about a relation already existing before I made this change. The unique constraint creates an index with the given name, and both tables and indexes share the same namespace.